azure-pipelines.yml

```diff
@@ -55,6 +55,10 @@ jobs:
     displayName: Install flake8
   - bash: flake8
     displayName: flake8 lint checks
+  - bash: python -m pip install black
```

Comment:
Can we just make this a separate job? The lint job takes like 20 seconds to warm up, so the difference in run-time will not be noticeable and it will be nice to see the status separately.